tests/std/tests/P0220R1_any/test.cpp

```diff
@@ -3220,4 +3241,5 @@ int main() {
     msvc::size_and_alignment::run_test();
     msvc::small_type::run_test();
     msvc::trivial::run_test();
+    msvc::gh_140_robust_against_adl::run_test();

```

Comment:
This is a test-compile test; in `run_test` I'm adding `globalMemCounter.disable_allocations = true / false;` to make it pass the runtime check. Is it ok to run this function here as with other run-time tests?